highlight_view.py

```diff
@@ -89,16 +89,17 @@ def prepare_data(errors):
     errors_augmented = []
     for error in errors:
         style = get_base_error_style(**error)
-        if not highlight_store.has_style(style):  # really?
-            continue
 
```

Comment:
That's important. Just remove it for now, and see if we actually get tracebacks from someone

Reply:
Agreed